frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/EditSoftwareModal/EditSoftwareModal.tsx

```diff
@@ -180,17 +187,33 @@ const EditSoftwareModal = ({
         },
       });
 
-      renderFlash(
-        "success",
-        <>
-          Successfully edited <b>{formData.software?.name}</b>.
-          {formData.selfService
-            ? " The end user can install from Fleet Desktop."
-            : ""}
-        </>
-      );
+      if ("title_id" in software && software.title_id && gitOpsModeEnabled) {
```

Comment:
I think the "title_id" in software is redundant. Is this checked with software.title_id?

Reply:
IIRC There's no `title_id` returned by `IAppStoreApp` so there's a type error. I'll see if I can refactor to be more explicit that software is `type ISoftwarePackage`